deploy/kubernetes/helm/che/custom-charts/che-devfile-registry/templates/configmap.yaml

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2018-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: che-devfile-registry
+data:
+{{- with .Values.cheDevfileImagesOverride -}}
+  {{- with .url }}
+  CHE_DEVFILE_IMAGES_REGISTRY_URL: {{ . | quote }}
```

Comment:
Does Che Server really respect this env var?

Even if yes, I still don't understand why Che Server needs to care about it.
Even without these changes, you should be able to override plugin and devfile registries images with:
```
--set che-devfile-registry.cheDevfileRegistry.image="non-existing"
```

```
> helm template \
  --set che-devfile-registry.cheDevfileRegistry.image="non-existing" \
  .                      
---
# Source: che/templates/serviceaccount.yaml
#
....
# SPDX-License-Identifier: EPL-2.0
#

apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: che
    component: devfile-registry
  name: devfile-registry
spec:
  replicas: 1
  revisionHistoryLimit: 2
  selector:
    matchLabels:
      app: che
      component: devfile-registry
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxSurge: 25%
      maxUnavailable: 25%
  template:
    metadata:
      labels:
        app: che
        component: devfile-registry
    spec:
      containers:
      - image: non-existing
        imagePullPolicy: Always
        name: che-devfile-registry
        ports:
        - containerPort: 8080